src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -286,6 +286,8 @@ public override bool IsExtensionMethod
             }
         }
 
+        internal bool IsStaticLocalFunction => _syntax.Modifiers.Any(SyntaxKind.StaticKeyword);
```

Comment:
>internal bool IsStaticLocalFunction => _syntax.Modifiers.Any(SyntaxKind.StaticKeyword); [](start = 8, length = 87)

Would IsStatic property work?

Reply:
`IsStatic` currently returns `true` always. See [https://github.com/dotnet/roslyn/issues/27719](https://github.com/dotnet/roslyn/issues/27719). We should replace this property when that issue is resolved.

---
In reply to: [244894319](https://github.com/dotnet/roslyn/pull/32067#discussion_r244894319) [](ancestors = 244894319)

Reply:
What would be the public API to answer the same question? It feels like we should have one at the time we ship this feature.

---
In reply to: [245043955](https://github.com/dotnet/roslyn/pull/32067#discussion_r245043955) [](ancestors = 245043955,244894319)